frontend/src/views/lessons/widgets/LessonSplash.jsx

```diff
@@ -1,12 +1,35 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Link, useParams } from 'react-router-dom'
+import { getData } from '../../../api/api'
 
 const LessonSplash = () => {
     const { lessonSlug } = useParams()
+    const [words, setWords] = useState(null);
+
+    useEffect(() => {
+        const getWord = async () => {
+            const result = await getData("http://localhost:8000/", `categories/words/${lessonSlug}`);

```

Comment:
`const result = await getData("http://localhost:8000/", `categories/words/${lessonSlug}`);`

Pls create an abstraction or `baseApi` so that you can resue the `baseURL`  and  `headers`
And create **env variable** for the `baseUrl` (your backend endpoint`

Here is the sample code related to this 
```js
import axios from 'axios';
import Cookies from 'js-cookie';

const TOKEN = Cookies.get('access_token');

const API = axios.create({
  baseURL: `${process.env.REACT_APP_SERVER_API}/api`,
  headers: {
    'Content-Type': 'application/json',
    'Authorization': `Bearer ${TOKEN}`,
  }
});

export default API;
```